content/docs/distribution/requisitions.md

```diff
@@ -78,27 +78,42 @@ You will then see the detail view of the requisition:
 
 ![Requisition Detail](/docs/distribution/images/requisition-detail.png)
 
+| Column        | Description                                                                                                                                                            |
+| :------------ | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Code**      | Code of the item                                                                                                                                                       |
+| **Name**      | Name of the item                                                                                                                                                       |
+| **Unit**      | The unit of measure for the item                                                                                                                                       |
+| **Our SOH**   | How much stock is available in this store                                                                                                                              |
+| **Their SOH** | How much stock is available in the customer's store                                                                                                                    |
+| **Requested** | The quantity requested by customer                                                                                                                                     |
+| **Supply**    | The quantity that will be supplied by this store                                                                                                                       |
```

Comment:
I feel like this is the piece that could be clearer... `The quantity planned to be supplied by this store`? `The quantity your store has committed to supply`?

Or even just adding `(could different from the quantity requested by the customer)` ?